config/locales/fr.yml

```diff
@@ -599,3 +599,5 @@ fr:
   price_graph: "Légende détail du prix"
   included_tax: "Inclut TVA"
   remove_tax: "Retirer TVA"
+  calculator_weight_per_kg: "Poids (par kg)"
```

Comment:
Am I understanding correctly that for the US instance to use Pounds instead of ounces and grams is going to require paying $850 a year (or more) to Transifex?